File: VitorBasso11611BCC034/programas tarefa 1/java/micro/micro01.java
public class micro01{
    public static void main(String[] args){
        double cel, far;
        System.out.println("Tabela de conversao: Celsius -> Fahrenheit");
        System.out.print("Digite a temperatura em Celsius: ");
        cel = Double.parseDouble(System.console().readLine());
        far = ((9 * cel) + 160) / 5;
        System.out.println("A nova temperatura e " + far + " F");
    }
}

File: trabalho 4/codigos/lua/micro/micro01.lua
function int main()
    local int cel
    local int far
    print(" tabela de conversao: Celsius -> Fahrenheit\n")
    print("Digite a temperatura em Celsius: ")
    cel = io.read()
    far = (9*cel + 160)/5
    print("A nova temperatura e: " ..far.." F")
end
